File: src/main/java/no/nav/service/pensjon/regler/to/BeregnInstitusjonsoppholdResponse.java
package no.nav.service.pensjon.regler.to;

import no.nav.domain.pensjon.regler.Pakkseddel;
import no.nav.domain.pensjon.regler.beregning.Beregning;
import no.nav.domain.pensjon.regler.beregning2011.AbstraktBeregningsResultat;
import no.nav.domain.pensjon.regler.beregning2011.BeregningsresultatUforetrygd;

/**
 * @author Nabil Safadi (Decisive) - PK-8518
 */
public class BeregnInstitusjonsoppholdResponse extends ServiceResponse {

	/**
	 * Pakkseddel for beregningene.
	 */
	private Pakkseddel pakkseddel = new Pakkseddel(true, true);
	
	// bruker1
	Beregning bruker1Beregning;
	AbstraktBeregningsResultat bruker1BeregningsResultat;
	Beregning bruker1BeregningTilRevurdering;
	AbstraktBeregningsResultat bruker1BeregningsResultatTilRevurdering;
    /* Bruker 1s beregningsresultatUforetrygd som skal utbetales ved institusjonsopphold */
    private BeregningsresultatUforetrygd bruker1BeregningsresultatUforetrygd;
    /* Bruker1s beregningsresultatUføretrgd som skal benyttes ved revurdering */
    private BeregningsresultatUforetrygd bruker1BeregningsresultatUforetrygdTilRevurdering;
	
	// bruker2
	Beregning bruker2Beregning;
	AbstraktBeregningsResultat bruker2BeregningsResultat;
	Beregning bruker2BeregningTilRevurdering;
	AbstraktBeregningsResultat bruker2BeregningsResultatTilRevurdering;
}

File: src/main/kotlin/no/nav/pensjon/regler/to/BeregnInstitusjonsoppholdResponse.kt
package no.nav.pensjon.regler.to


import no.nav.pensjon.regler.domain.beregning.Beregning
import no.nav.pensjon.regler.domain.beregning2011.AbstraktBeregningsResultat
import no.nav.pensjon.regler.domain.beregning2011.BeregningsresultatUforetrygd


class BeregnInstitusjonsoppholdResponse : ServiceResponse() {
    // bruker1
    var bruker1Beregning: Beregning? = null
    var bruker1BeregningsResultat: AbstraktBeregningsResultat? = null
    var bruker1BeregningTilRevurdering: Beregning? = null
    var bruker1BeregningsResultatTilRevurdering: AbstraktBeregningsResultat? = null

    /* Bruker 1s beregningsresultatUforetrygd som skal utbetales ved institusjonsopphold */
    var bruker1BeregningsresultatUforetrygd: BeregningsresultatUforetrygd? = null

    /* Bruker1s beregningsresultatUføretrgd som skal benyttes ved revurdering */
    var bruker1BeregningsresultatUforetrygdTilRevurdering: BeregningsresultatUforetrygd? = null

    // bruker2
    var bruker2Beregning: Beregning? = null
    var bruker2BeregningsResultat: AbstraktBeregningsResultat? = null
    var bruker2BeregningTilRevurdering: Beregning? = null
    var bruker2BeregningsResultatTilRevurdering: AbstraktBeregningsResultat? = null
}